surveyor/demo/website_menu.xml

```diff
@@ -6,7 +6,7 @@
         <field name="url">/our-services</field>
         <field name="parent_id" model="website.menu" eval="obj().search([
             ('website_id', '=', ref('website.default_website')),
-            ('url', '=', '/default-main-menu'),
+            ('url', '=', '#'),
```

Comment:
surveyor seems to be the only one in the list where the menu was present before your change. Could you investigate why it is not needed?